test-harness/src/test/java/com/netflix/conductor/tests/utils/WorkflowCleanUpUtil.java

```diff
@@ -0,0 +1,54 @@
+package com.netflix.conductor.tests.utils;
+
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.core.execution.WorkflowExecutor;
+import com.netflix.conductor.dao.QueueDAO;
+import com.netflix.conductor.service.ExecutionService;
+import com.netflix.conductor.service.MetadataService;
+import org.apache.commons.lang.StringUtils;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.io.FileOutputStream;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Singleton
+public class WorkflowCleanUpUtil {
```

Comment:
Nit: Trying to understand what this class does. JavaDoc?

Reply:
Moved to another utility class : WorkflowTestUtil and added documentation